### PR TITLE
Fix Contributing.md

### PR DESCRIPTION
**Pull request by vidorteg**
Fixing some old paths in the Contributing.md file.

### CODE REVIEW

**Comment by vidorteg** [comment]
@codepo8 could you help me with a quick review on this one so we can address #124?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,13 +18,12 @@ Contributions are always welcome! We only ask that you open an issue first so we
 * Clone this repo
 * Run `npm install` in '/vscode-edge-devtools'
 * On an administrator prompt execute the following commands (assuming your drive is located at C:\)
-  * `mkdir c:\edgeoss\code`
   * `mkdir c:\edgeoss\src\out\Release\gen\devtools`
-  * `mkdir c:\edgeoss\src\third_party\blink\renderer\devtools\front_end`
+  * `mkdir c:\edgeoss\src\third_party\devtools-frontend\src\front_end`
```
Comment (mwjacksonmsft) [inline CONTRIBUTING.md]: Can we also fix the references to edgeoss since we prefer just edge now?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,17 +18,16 @@ Contributions are always welcome! We only ask that you open an issue first so we
 * Clone this repo
 * Run `npm install` in '/vscode-edge-devtools'
 * On an administrator prompt execute the following commands (assuming your drive is located at C:\)
-  * `mkdir c:\edgeoss\code`
```
Comment (mwjacksonmsft) [inline CONTRIBUTING.md]: I assume we no longer need this line?

Comment (vidorteg) [reply CONTRIBUTING.md]: yep, not needed anymore

**Pull request merged**